README.md

```diff
@@ -76,6 +76,43 @@ $ remme account get-balance \
 368440.0
 ```
 
+### Blockchain info
```

Comment:
When you operate some libraries where you see `blockchain info`, it is ok.
When you provide the independent layer of user experience, you should care about users in own way.

I mean, In the library it is blockchain information, but CLI user expects section batch.

Reply:
Done.